src/features/code_actions.zig

```diff
@@ -109,7 +109,35 @@ fn handleUnusedFunctionParameter(builder: *Builder, actions: *std.ArrayListUnman
 
     const block = node_datas[payload.func].rhs;
 
-    const new_text = try createDiscardText(builder, identifier_name, token_starts[node_tokens[payload.func]], true);
+    // If we are on the "last parameter" that requires a discard, then we need to append a newline,
+    // as well as any relevant indentations, such that the next line is indented to the same column.
+    // To do this, you may have a function like:
+    // fn(a: i32, b: i32, c: i32) void { _ = a; _ = b; _ = c; }
+    // or
+    // fn(
+    //     a: i32,
+    //     b: i32,
+    //     c: i32,
+    // ) void { ... }
+    // We have to be able to detect both cases.
+    var param_end = offsets.tokenToLoc(tree, ast.paramLastToken(tree, payload.get(tree).?)).end;
```

Comment:
Further towards the end of this function `payload.get(tree)` is being called again. I would suggest to do it once and store into a new variable.